### ISSUE TITLE
API: Move Mainfest to root

### ISSUE THREAD

**Issue by LIN810116** [SPARC-FAIR-Codeathon/sparc-me#88]
Update the add_datqa function

Relates to #80 #84

**Comment by LinkunGao** [comment]
- Add a new function in dataset class: 
   ```python
     def add_data(self, source_path, data_type, subject, sample, sds_parent_dir=None, copy=True, overwrite=False, sample_metadata={}, subject_metadata={}):
   ```
  - use `dataset.add_data( data_type='primary')` to call `add_primary_data`
  - use  `dataset.add_data( data_type='derivative')` to call `add_derivative_data`
- Modified derivative_folder variable in `add_primary_data` and `add_derivative_data`:
   -  use  derivative_folder_list to instead. 
   - before derivative_folder = '/tem/data/primary/subject/sample'
   - now derivative_folder_list = ['tem/data', 'primary', 'subject', 'sample' ]
   -  the benefits for this is we can easily to combine different paths in other functions, in order to we can save manifest.xlsx in root folder.
- all test on dataset version 2.0.0

**Comment by LIN810116** [comment]
Resolved by #89
